var/spack/repos/builtin/packages/flux-pmix/package.py

```diff
@@ -18,10 +18,11 @@ class FluxPmix(AutotoolsPackage):
     maintainers("grondo")
 
     version("main", branch="main")
+    version("0.4.0", sha256="f7f58891fc9d9a97a0399b3ab186f2cae30a75806ba0b4d4c1307f07b3f6d1bc")
     version("0.3.0", sha256="88edb2afaeb6058b56ff915105a36972acc0d83204cff7f4a4d2f65a5dee9d34")
     version("0.2.0", sha256="d09f1fe6ffe54f83be4677e1e727640521d8110090515d94013eba0f58216934")
 
-    depends_on("flux-core@0.49.0:", when="@0.3.0:")
+    depends_on("flux-core@0.49.0", when="@0.3:")
```

Comment:
@vsoch I think you basically had this correct before.  From what I read in @grondo's comments, I believe you want to restore the `flux-core` version range:

```suggestion
    depends_on("flux-core@0.49.0:", when="@0.3:")
```

Reply:
ahh now we've come full circle! Simba, is that you? :lion: